### ISSUE TITLE
current php images fail to build because remote script change

### ISSUE THREAD

**Issue by keithbentrup** [magento/magento-cloud-docker#331]
### Steps to reproduce

1.  Try to build any php image that references https://deb.nodesource.com/setup_10.x. That includes the current dev branch and the latest tagged version (1.2.4).
2. Since the current setup_10.x fails to properly install npm, `npm install -g grunt-cli` will fail and the build will not complete.

### Expected result

1. Successfully built php images.

### Actual result

1. npm not found and image build stops
```bash
Step 18/45 : RUN npm install -g grunt-cli
 ---> Running in 1b1a0bfaa66c
/bin/sh: 1: npm: not found
```

changing `https://deb.nodesource.com/setup_10.x` to `https://deb.nodesource.com/setup_12.x` will work, but I don't know what might require node v10.x.

**Comment by YPyltiai** [comment]
@BaDos , please see this issue.

**Comment by andriyShevtsov** [comment]
Created task ACMP-1539

**Comment by BaDos** [comment]
@keithbentrup it's fixed in 1.3.0

**Comment by YPyltiai** [comment]
@keithbentrup , could you please confirm the issue is resolved for you?
